picard/file.py

```diff
@@ -158,6 +158,28 @@ def _load(self, filename):
         """Load metadata from the file."""
         raise NotImplementedError
 
+    def _test_same(self):
+        if config.setting['clear_existing_tags']:
+            return False
+        new_metadata = self.new_metadata
+        orig_metadata = self.orig_metadata
+        tags = set(new_metadata.keys() + orig_metadata.keys())
+        for name in filter(lambda x: x.startswith("~config:"), tags):
+            new_metadata[name] = config.setting[name.split(':')[1]]
+            if new_metadata[name] != orig_metadata[name]:
+                return False
+        for name in filter(lambda x: (not x.startswith("~")
+                                      or x.startswith("~id3")
```

Comment:
I dont think we need to add a check specifically for id3 tags since, we are already 'AND' the above result with self.supports_tag